po/it.po

```diff
@@ -0,0 +1,264 @@
+msgid "File Name"
+msgstr "Nome File"
+
+#: src/Views/EditView.vala:36
+msgid ""
+"Name of the file where these app info is saved. Only lowercase letters and "
+"numbers are allowed."
+msgstr ""
+"Nome del file in cui vengono salvate queste informazioni sull'app. Solo lettere minuscole e "
+"i numeri sono ammessi."
+
+#: src/Views/EditView.vala:42
+msgid "Suffix of the file"
+msgstr "Suffisso del file"
+
+#: src/Views/EditView.vala:53
+msgid "App Name"
+msgstr "Nome App"
+
+#: src/Views/EditView.vala:54
+msgid "Shown in the launcher or Dock."
+msgstr "Mostrato nel launcher o Dock."
+
+#: src/Views/EditView.vala:65
+msgid "Comment"
+msgstr "Commento"
+
+#: src/Views/EditView.vala:66
+msgid "A tooltip text to describe what the app helps you to do."
+msgstr "Un testo di descrizione per descrivere cosa ti aiuta a fare l'app."
+
+#: src/Views/EditView.vala:77
+msgid "Exec File"
+msgstr "File esecutivo"
+
+#: src/Views/EditView.vala:79
+msgid ""
+"The command/app launched when you click the app entry in the launcher. "
+"Specify in an absolute path or an app's alias name."
+msgstr ""
+"Il comando/l'app è avviata quando si fa clic sulla voce dell'app nel programma di avvio. "
+"Specificare in un percorso assoluto o il nome alias di un'app."
+
+#: src/Views/EditView.vala:92
+msgid "Icon File"
+msgstr "Icona File"
+
+#: src/Views/EditView.vala:94
+msgid ""
+"The icon branding the app. Specify in an absolute path or an icon's alias "
+"name."
+msgstr ""
+"L'icona che contraddistingue l'app. Specifica in un percorso assoluto il nome o l'alias "
+"dell'icona."
+
+#: src/Views/EditView.vala:107
+msgid "App Categories"
+msgstr "Categorie di app"
+
+#: src/Views/EditView.vala:108
+msgid "Type of the app, multiplly selectable."
+msgstr "Tipo di app, selezionabile in modo multiplo."
+
+#: src/Views/EditView.vala:117
+msgid "Run in Terminal"
+msgstr "Esegui nel terminale"
+
+#: src/Views/EditView.vala:120
+msgid "Check this in if you want to registar a CUI app."
+msgstr "Controlla questo se vuoi registrare un'app CUI."
+
+#: src/Views/EditView.vala:122
+msgid "Save entry"
+msgstr "Salva voce"
+
+#: src/Views/EditView.vala:147
+msgid "Select an executable file"
+msgstr "Seleziona un file eseguibile"
+
+#: src/Views/EditView.vala:148 src/Views/EditView.vala:174
+msgid "Open"
+msgstr "Apri"
+
+#: src/Views/EditView.vala:148 src/Views/EditView.vala:174
+#: src/Views/FilesView.vala:108
+msgid "Cancel"
+msgstr "Cancella"
+
+#: src/Views/EditView.vala:170
+msgid "PNG, SVG, or XMP files"
+msgstr "File PNG, SVG o XMP"
+
+#: src/Views/EditView.vala:173
+msgid "Select an icon file"
+msgstr "Seleziona un file icona"
+
+#: src/Views/FilesView.vala:23
+msgid "No valid app entries found"
+msgstr "Nessuna voce valida per l'app trovata"
+
+#: src/Views/FilesView.vala:24
+msgid ""
+"If you've never created one, go back to Welcome View and click “New Entry”."
+msgstr ""
+"Se non ne hai mai creato uno, torna alla visualizzazione di benvenuto e fai clic su "Nuova voce"."
```

Comment:
```suggestion
"Se non ne hai mai creato uno, torna alla visualizzazione di benvenuto e fai clic su “Nuova voce”."
```

The source string uses `”`, not `"`. A tiny thing but this causes the build error